cluster/ubuntu/config-default.sh

```diff
@@ -34,6 +34,15 @@ export SERVICE_CLUSTER_IP_RANGE=${SERVICE_CLUSTER_IP_RANGE:-192.168.3.0/24}  # f
 # define the IP range used for flannel overlay network, should not conflict with above SERVICE_CLUSTER_IP_RANGE
 export FLANNEL_NET=${FLANNEL_NET:-172.16.0.0/16}
 
+# Optionally add other contents to the Flannel configuration JSON
+# object normally stored in etcd as /coreos.com/network/config.  Use
+# JSON syntax suitable for insertion into a JSON object constructor
+# after other field name:value pairs.  For example:
+# FLANNEL_OTHER_NET_CONFIG=', "SubnetMin": "172.16.10.0", "SubnetMax": "172.16.90.0"'
+
+export FLANNEL_OTHER_NET_CONFIG
```

Comment:
To clarify, I wasn't asking why you export separately from setting it, which seems fine, but why you do export *before* setting it (in the un-commented-out version). To be more concrete, shouldn't the un-commented-out `FLANNEL_OTHER_NET_CONFIG=''` line go before the `export FLANNEL_OTHER_NET_CONFIG` line?